autogen/agentchat/contrib/multimodal_web_surfer/multimodal_web_surfer.py

```diff
@@ -117,12 +117,22 @@ def __init__(
 
         # Create the context -- are we launching a persistent instance?
         if chromium_data_dir is None:
-            browser = self._playwright.chromium.launch(**launch_args)
+            if chromium_channel == "chromium":
+                browser = self._playwright.chromium.launch(**launch_args)
+            elif chromium_channel == "firefox":
```

Comment:
I wonder if we should rename this variable if applied to Firefox. It is no longer just the chromium channel.

Reply:
I wasn't sure of how many other variations of 'scenario.py' exist (and maybe in other prs) so for now preferred not to break the arg. I can check this out though.